src/GitHubParserHook.php

```diff
@@ -22,6 +23,15 @@ class GitHubParserHook implements HookHandler {
 	private $repoName;
 	private $branchName;
 
+	// Parameters for SyntaxHighlight extension (formerly SyntaxHighlight_GeSHi)
+	private $lang;
+	private $line;
+	private $start;
+	private $highlight;
+	private $inline;
+
+	private static $syntaxHighlightFields = array('lang', 'line', 'start', 'highlight', 'inline');
```

Comment:
You could use `[]`. Also note that there should be spaces between the brackets and values